turborepo-tests/integration/fixtures/turbo-configs/interactive.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": ["foo.txt"],
+  "globalEnv": ["SOME_ENV_VAR"],
+  "pipeline": {
+    "build": {
+      "env": ["NODE_ENV"],
+      "interactive": true,
+      "outputs": []
+    },
+    "my-app#build": {
+      "outputs": ["banana.txt", "apple.json"],
+      "dotEnv": [".env.local"]
+    },
+
+    "something": {},
+    "//#something": {},
+
+    "maybefails": {}
+  }
```

we should make fixtures as minimal as possible!

```suggestion
  "pipeline": {
    "build": {
      "interactive": true
    }
  }
```